src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,313 @@
+        secondParam = SECOND_POSITIVE_VALUE;
+        double expected = firstParam - secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '-');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void subtractionZeroAtB() {
+        firstParam = FIRST_POSITIVE_VALUE;
+        secondParam = ZERO_VALUE;
+        double expected = firstParam - secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '-');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void subtractionMinAndMaxValues() {
+        firstParam = Integer.MAX_VALUE;
+        secondParam = Integer.MIN_VALUE;
+        double expected = firstParam - secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '-');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void multiplicationTwoPositive() {
+        firstParam = FIRST_POSITIVE_VALUE;
+        secondParam = SECOND_POSITIVE_VALUE;
+        double expected = firstParam * secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '*');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void multiplicationTwoNegative() {
+        firstParam = FIRST_NEGATIVE_VALUE;
+        secondParam = SECOND_NEGATIVE_VALUE;
+        double expected = firstParam * secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '*');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void multiplicationPositiveAndNegative() {
+        firstParam = FIRST_NEGATIVE_VALUE;
+        secondParam = SECOND_POSITIVE_VALUE;
+        double expected = firstParam * secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '*');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void multiplicationZeroAtA() {
+        firstParam = ZERO_VALUE;
+        secondParam = SECOND_POSITIVE_VALUE;
+        double expected = firstParam * secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '*');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void multiplicationZeroAtB() {
+        firstParam = FIRST_POSITIVE_VALUE;
+        secondParam = ZERO_VALUE;
+        double expected = firstParam * secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '*');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void multiplicationMinAndMaxValues() {
+        firstParam = Integer.MAX_VALUE;
+        secondParam = Integer.MIN_VALUE;
+        double expected = firstParam * secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '*');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void divisionTwoPositive() {
+        firstParam = FIRST_POSITIVE_VALUE;
+        secondParam = SECOND_POSITIVE_VALUE;
+        double expected = firstParam / secondParam;
```

Comment:
same for all division tests and in solution

```suggestion
        double expected = (double) firstParam / secondParam;
```